### PR TITLE
group theory

### PR DESCRIPTION
**Pull request by gerhean**

### CODE REVIEW

**Review by qjqqyy — APPROVED** [review]
lgtm

File: examples/groups.coc [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+			(B (B b a) (B b b))
+			(B b a)			
+			(lrassoci (B b a) b b)
+			
+			(orderTwo b (B b a))
+			)))))
+))))))));
+
+/*
+define rIdAndrInvImpliesUniqueId =
```
Comment (thomastanck) [inline examples/groups.coc]: Remove commented code

**Review by thomastanck — CHANGES_REQUESTED** [review]
Could you add tests to the Makefile under `test:`?

Also, your files have a lot of trailing spaces. could you quickly get rid of them as well?

File: examples/groups.coc (hunk 1 above)
Comment (gerhean) [reply examples/groups.coc]: done

**Review by thomastanck — APPROVED** [review]
Very nice. Good job!

**Pull request merged**